packages/client/src/external/listenForDocumentQueries.ts

```diff
@@ -1,32 +1,13 @@
-import { GraphQLFormattedError } from 'graphql';
 import { getExtensionId } from '@finch-graphql/browser-polyfill';
 import { queryApi } from '../client';
 import { FinchQueryOptions } from '@finch-graphql/types';
 import { FinchDocumentEventNames, FinchRequestEvent } from './types';
+import { sendResponse } from './listenForExternalRequests';
 
 /**
- * sendResponse a method to send a response through the document.
- * @param event the finch request event
- * @param data the data from the graphQL response
- * @param errors the errors from the graphQL response
- */
-export const sendResponse = (
-  requestId: string,
-  data: any,
-  errors: Array<GraphQLFormattedError>,
-) => {
-  window.postMessage(
-    {
-      type: FinchDocumentEventNames.Response,
-      data,
-      errors,
-      requestId,
-    },
-    '*',
-  );
-};
-
-/**
+ * `DEPRECATED` use `listenForExternalRequests` instead, this method
```

Comment:
opted to do this instead of `@deprecated`

![Screen Shot 2022-01-25 at 6 09 01 PM](https://user-images.githubusercontent.com/578259/151092590-a01b8111-b8d7-4f56-b504-5f0228890d80.png)

Reply:
Why did you opt for this approach? Was `@deprecated` not as apparent when someone views it?

Reply:
Yeah, no additional styling just looked exactly how the params from jsdoc's looked.